src/main/webapp/app/course/plagiarism-cases/student-view/detail-view/plagiarism-case-student-detail-view.component.ts

```diff
@@ -35,6 +43,9 @@ export class PlagiarismCaseStudentDetailViewComponent implements OnInit, OnDestr
             this.plagiarismCasesService.getPlagiarismCaseDetailForStudent(this.courseId, this.plagiarismCaseId).subscribe({
                 next: (res: HttpResponse<PlagiarismCase>) => {
                     this.plagiarismCase = res.body!;
+                    this.exerciseTitle =
+                        this.plagiarismCase.exercise!.title!.length > 40 ? this.plagiarismCase.exercise?.title?.slice(0, 35) + '…' : this.plagiarismCase.exercise?.title;
```

Comment:
Why didn't you use the abbreviate helper method here?

Reply:
That was an oversight and can be removed, we will use the proposed CSS solution here.